[2022-02-19 06:34:02,126] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: demo_etl.Load_Data.Save_to_DB manual__2022-02-19T06:33:48.458432+00:00 [queued]>
[2022-02-19 06:34:02,281] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: demo_etl.Load_Data.Save_to_DB manual__2022-02-19T06:33:48.458432+00:00 [queued]>
[2022-02-19 06:34:02,282] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 06:34:02,282] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-19 06:34:02,282] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 06:34:02,444] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): Load_Data.Save_to_DB> on 2022-02-19 06:33:48.458432+00:00
[2022-02-19 06:34:02,622] {standard_task_runner.py:52} INFO - Started process 34787 to run task
[2022-02-19 06:34:03,427] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'demo_etl', 'Load_Data.Save_to_DB', 'manual__2022-02-19T06:33:48.458432+00:00', '--job-id', '268', '--raw', '--subdir', 'DAGS_FOLDER/demo_etl.py', '--cfg-path', '/tmp/tmpgjcwfcue', '--error-file', '/tmp/tmpz93lpb92']
[2022-02-19 06:34:03,428] {standard_task_runner.py:77} INFO - Job 268: Subtask Load_Data.Save_to_DB
[2022-02-19 06:34:09,274] {logging_mixin.py:109} INFO - Running <TaskInstance: demo_etl.Load_Data.Save_to_DB manual__2022-02-19T06:33:48.458432+00:00 [running]> on host c39e331ab017
[2022-02-19 06:34:11,846] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=demo_etl
AIRFLOW_CTX_TASK_ID=Load_Data.Save_to_DB
AIRFLOW_CTX_EXECUTION_DATE=2022-02-19T06:33:48.458432+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-19T06:33:48.458432+00:00
[2022-02-19 06:34:11,879] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-02-19 06:34:11,896] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'next_ds' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds }}' instead.
[2022-02-19 06:34:11,933] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'next_ds_nodash' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds_nodash }}' instead.
[2022-02-19 06:34:11,976] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'next_execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_end' instead.
[2022-02-19 06:34:12,281] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'prev_ds' from the template is deprecated and will be removed in a future version.
[2022-02-19 06:34:12,405] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'prev_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-02-19 06:34:12,417] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'prev_execution_date' from the template is deprecated and will be removed in a future version.
[2022-02-19 06:34:12,418] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'prev_execution_date_success' from the template is deprecated and will be removed in a future version. Please use 'prev_data_interval_start_success' instead.
[2022-02-19 06:34:12,419] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'tomorrow_ds' from the template is deprecated and will be removed in a future version.
[2022-02-19 06:34:12,424] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'tomorrow_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-02-19 06:34:12,429] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'yesterday_ds' from the template is deprecated and will be removed in a future version.
[2022-02-19 06:34:12,440] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'yesterday_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-02-19 06:34:14,227] {logging_mixin.py:109} INFO - Done !!
[2022-02-19 06:34:17,989] {logging_mixin.py:109} INFO - Done !!
[2022-02-19 06:34:18,479] {logging_mixin.py:109} INFO -       VendorID lpep_pickup_datetime  ... trip_type  congestion_surcharge
0            2  2019-12-18 15:52:30  ...         1                     0
1            2  2020-01-01 00:45:58  ...         2                     0
2            2  2020-01-01 00:41:38  ...         1                     0
3            1  2020-01-01 00:52:46  ...         1                     2
4            1  2020-01-01 00:19:57  ...         1                     0
...        ...                  ...  ...       ...                   ...
1995         2  2021-01-02 12:48:43  ...         1                     0
1996         1  2021-01-02 12:16:47  ...         1                     0
1997         2  2021-01-02 12:07:17  ...         1                     0
1998         2  2021-01-02 12:32:45  ...         1                     0
1999         2  2021-01-02 12:02:46  ...         1                     0

[2000 rows x 18 columns]
[2022-02-19 06:34:35,631] {logging_mixin.py:109} INFO - Save Data to Database Done !!
[2022-02-19 06:34:35,631] {python.py:175} INFO - Done. Returned value was: None
[2022-02-19 06:34:35,643] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=demo_etl, task_id=Load_Data.Save_to_DB, execution_date=20220219T063348, start_date=20220219T063402, end_date=20220219T063435
[2022-02-19 06:34:35,700] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-19 06:34:35,737] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
